node/task.ts

```diff
@@ -614,20 +597,38 @@ export const checkPath = im._checkPath;
  */
 export function cd(path: string): void {
     if (path) {
-        shell.cd(path);
-        _checkShell('cd');
+        let errMsg: string = '';
+        if (!fs.existsSync(path)) {
+            errMsg = loc('LIB_OperationFailed', 'cd ' + path, 'No such file or directory');
+        }
+        else if (!fs.statSync(path).isDirectory()) {
+            errMsg = loc('LIB_OperationFailed', 'cd ' + path, 'Not a directory');
```

Comment:
consider: why present the error message to the user as a command line instruction? The `cd` command is not actually being invoked.